orders-war/src/test/java/au/gov/qld/pub/orders/service/AttachmentServiceTest.java

```diff
@@ -125,165 +117,255 @@ private void setDefaultReturns(Item item, String suffix) {
 //        when(item.isPaid()).thenReturn(true); //UnnecessaryStubbingException
         when(item.getQuantityPaid()).thenReturn(QUANTITY_PAID);
 	}
-    
+
     @Test
     public void sendOrderAndItemDetailsToFormServiceAndReturnAsAttachmentsForBusiness() throws Exception {
-        when(client.execute(argThat(postRequest(BUSINESS_FORM_URI, EXPECTED_FORM_DATA))))
+        when(businessResponse.getCode()).thenReturn(500, AttachmentService.OKAY_STATUS_CODE);
+        when(businessResponse.getReasonPhrase()).thenReturn("500", "200");
+        when(businessResponse.getEntity()).thenReturn(new StringEntity(BUSINESS_CONTENT));
+        when(customerResponse.getCode()).thenReturn(500, AttachmentService.OKAY_STATUS_CODE);
+        when(customerResponse.getReasonPhrase()).thenReturn("500", "200");
+        when(customerResponse.getEntity()).thenReturn(new StringEntity(CUSTOMER_CONTENT));
+        when(bundledCustomerResponse.getCode()).thenReturn(500, AttachmentService.OKAY_STATUS_CODE);
+        when(bundledCustomerResponse.getReasonPhrase()).thenReturn("500", "200");
+        when(bundledCustomerResponse.getEntity()).thenReturn(new StringEntity(BUNDLED_CUSTOMER_CONTENT));
```

Comment:
While I assume not every test needs this if it's been removed from `setUp()`, it does still appear to be repeated a few times, so should this ideally be its own function?

Reply:
was having major issues with this test since the http5 client changed our response status worked.

Reply:
fixed up now